frameworks/Clojure/duct/benchmark_config.json

```diff
@@ -0,0 +1,115 @@
+{
+  "framework": "duct",
+  "tests": [
+    {
+      "default": {
+        "json_url": "/json",
+        "plaintext_url": "/plaintext",
+        "db_url": "/db",
+        "query_url": "/queries?queries=",
+        "fortune_url": "/fortunes",
+        "port": 3000,
+        "approach": "Realistic",
+        "classification": "Micro",
+        "database": "Postgres",
+        "framework": "duct",
+        "language": "Clojure",
+        "flavor": "None",
+        "orm": "Raw",
+        "platform": "Servlet",
```

Comment:
Just checking - is `"platform": "Servlet"` correct?  I don't see a servlet container such as Resin or Tomcat being used in the dockerfiles to launch the web application, so I'm wondering if this part is intentional.

Reply:
You're right, the platform for the `default`, `mongodb` and `httpkit` is supposed to be `Ring`. I also updated the other implementations like `aleph` -> `Netty`, `immutant` -> `Undertow`.